build.gradle

```diff
@@ -19,10 +9,10 @@ buildscript {
 }
 
 plugins {
-    // Apply the java plugin to add support for Java
     id 'java'
-    // Apply the application plugin to add support for building a CLI application
     id 'application'
+    id 'jacoco'
+    id 'org.sonarqube' version '3.3'
     id 'com.palantir.docker' version '0.26.0'
     id 'com.github.johnrengelman.shadow' version '6.1.0'
```

Comment:
Are `com.github.jengelman.gradle.plugins:shadow:4.0.4` and `id 'com.github.johnrengelman.shadow' version '6.1.0'` 2 versions of the same plugin ?
If true, the entire `buildscript` block should be removable.

Reply:
True, I updated bunch of lines for that, thanks :)